### PR TITLE
Add ErrorCodeLogLevelConfig to control error code log levels

### PR DESCRIPTION
**Pull request by ryanhall07**
This allows applications to use different log levels in different
environments.

### CODE REVIEW

File: misk/src/test/kotlin/misk/config/ConfigTest.kt [hunk 1]
```diff
@@ -43,6 +45,9 @@ class ConfigTest {
         assertThat(testConfig.consumer_a).isEqualTo(ConsumerConfig(0, 1))
         assertThat(testConfig.consumer_b).isEqualTo(ConsumerConfig(1, 2))
         assertThat(testConfig.duration).isEqualTo(DurationConfig(Duration.parse("PT1S")))
+        assertThat((testConfig.error_code_log_level)).isEqualTo(
+            ErrorCodeLogLevelConfig(Level.INFO, Level.ERROR)
+        )
```
Comment (ryanhall07) [inline misk/src/test/kotlin/misk/config/ConfigTest.kt]: this looks weird to me, but i'm using the SQ formatter

**Comment by mmihic** [comment]
I think we want this to be a little more general purpose. ``ExceptionMapper``s can be used to map ay exception to a response, not just ``ActionException``s, so it should be possible to control whether to log other types of exceptions converted by ``ExceptionMapper``s

**Comment by ryanhall07** [comment]
@mmihic yea this was just to allow config overrides for only the `ActionExceptionMapper`. maybe a better name for the config value? `action_exception_log_levels:` ?

the more generic exception handling is in https://github.com/square/misk/pull/108

**Comment by ryanhall07** [comment]
@mmihic updated this to be a specific `ActionExceptionLogLevelConfig` for configuring the `ActionExceptionMapper`. PTAL

**Comment by swankjesse** [comment]
Reformatted:
https://github.com/square/misk/pull/116

**Pull request closed without merging**